core/src/test/scala/io/finch/syntax/MapperSyntaxBehaviour.scala

```diff
@@ -0,0 +1,104 @@
+package io.finch.syntax
+
+import cats.syntax.functor._
+import com.twitter.finagle.http.Response
+import io.finch._
+import io.finch.{FinchSpec, Text}
+import org.scalacheck.Arbitrary
+import org.scalatest.prop.GeneratorDrivenPropertyChecks
+
+trait MapperSyntaxBehaviour extends FinchSpec with GeneratorDrivenPropertyChecks {
```

Comment:
I think we name these `XSpec`?

Reply:
It's not a `Spec` strictly speaking, it's a trait with behaviour method

Reply:
Sure. But we have zero tests named `Behaviour` and something like 30 tests named `Spec`. Maybe we can rename it for the sake of consistency?